test/test_helper.rb

```diff
@@ -0,0 +1,18 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+require 'rails/test_help'
+
+class ActiveSupport::TestCase
+  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
```

Comment:
__[rubocop]__
* Line is too long. [82/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->